common/utils.cpp

```diff
@@ -685,5 +706,37 @@ std::string getCurrentSystemTime()
     return ss.str();
 }
 
+std::string getLocationObject(std::string locationCode,
```

Comment:
this cannot be in generic code ..move this to oem.

This is a very specific use case for ibm (i dont think any one would have this kind of requirement)

Reply:
Moved this function to oem/ibm